.pre-commit-config.yaml

```
@@ -5,12 +5,12 @@ repos:
     -   id: check-yaml
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
--   repo: https://github.com/psf/black
-    rev: 24.3.0
-    hooks:
-    -   id: black
 - repo: https://github.com/charliermarsh/ruff-pre-commit
-  # Ruff version.
-  rev: 'v0.3.4'
+  rev: 'v0.9.7'
   hooks:
     - id: ruff
+      args: ["--fix"]
```

Let's also format the code with ruff
```suggestion
      args: ["--fix"]
    - id: ruff-format
```